vars/provisionNodes.groovy

```diff
@@ -163,239 +126,180 @@ def call(Map config = [:]) {
-    def provision_script = '''set -ex
-                              rm -f ci_key*
-                              ssh-keygen -N "" -f ci_key
-                              cat << "EOF" > ci_key_ssh_config
+  // Prepare the node for daos/cart testing
+  def provision_script = '''set -ex
+                            rm -f ci_key*
+                            ssh-keygen -N "" -f ci_key
+                            cat << "EOF" > ci_key_ssh_config
 host wolf-*
     CheckHostIp no
     StrictHostKeyChecking no
     UserKnownHostsFile /dev/null
     LogLevel error
 EOF'''
-    provision_script += '\nclush -l root -w ' + nodeString + ' -c' +
-                      ''' ci_key* --dest=/tmp/
-                          clush -B -S -l root -w ''' + nodeString +
-                      ''' "set -ex
-                          env > /root/last_run-env.txt
-                          my_uid=''' + env.UID + '''
-                          if ! grep \\":\\$my_uid:\\" /etc/group; then
-                            groupadd -g \\$my_uid jenkins
-                          fi
-                          mkdir -p /localhome
-                          if ! grep \\":\\$my_uid:\\$my_uid:\\" /etc/passwd; then
-                            useradd -b /localhome -g \\$my_uid -u \\$my_uid jenkins
-                          fi
-                          mkdir -p /localhome/jenkins/.ssh
-                          cat /tmp/ci_key.pub >> /localhome/jenkins/.ssh/authorized_keys
-                          cat /tmp/ci_key.pub >> /root/.ssh/authorized_keys
-                          mv /tmp/ci_key.pub /localhome/jenkins/.ssh/id_rsa.pub
-                          mv /tmp/ci_key /localhome/jenkins/.ssh/id_rsa
-                          mv /tmp/ci_key_ssh_config /localhome/jenkins/.ssh/config
-                          chmod 700 /localhome/jenkins/.ssh
-                          chmod 600 /localhome/jenkins/.ssh/{authorized_keys,id_rsa*,config}
-                          chown -R jenkins.jenkins /localhome/jenkins/.ssh
-                          echo \\"jenkins ALL=(ALL) NOPASSWD: ALL\\" > /etc/sudoers.d/jenkins'''
-    def iterate_repos = '''for repo in ''' + inst_repos + '''; do
-                               branch=\\"master\\"
-                               build_number=\\"lastSuccessfulBuild\\"
-                               if [[ \\\$repo = *@* ]]; then
-                                   branch=\\"\\\${repo#*@}\\"
-                                   repo=\\"\\\${repo%@*}\\"
-                                   if [[ \\\$branch = *:* ]]; then
-                                       build_number=\\"\\\${branch#*:}\\"
-                                       branch=\\"\\\${branch%:*}\\"
-                                   fi
-                               fi'''
-    if (distro.startsWith("el7")) {
-       // Since we don't have CORCI-711 yet, erase things we know could have
-       // been put on the node previously
+  provision_script += '\nclush -B -l root -w ' + nodeString + ' -c' +
```

Comment:
Any reason to end the previous `provision_script` assignment on line 142 and then just append to it here?  Why not just keep it going in the same `'''`?